package.json

```diff
@@ -12,6 +12,7 @@
     "express-ejs-layouts": "^2.5.0",
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.10.6",
+    "nodeman": "^1.1.2",
```

Comment:
Did you include this? I believe it's spelt "nodemon", with an "o". You shouldn't need to do this as nodemon is already in the package.json file under devDependencies.